quiche/src/lib.rs

```diff
@@ -742,6 +751,26 @@ impl Config {
         Self::with_tls_ctx(version, tls::Context::new()?)
     }
 
+    /// Creates a default config object using commonly used values
+    /// Useful in testing
+    pub fn default() -> Result<Self> {
```

Comment:
We shouldn't do this, there is no sensible default for the QUIC transport because the values depend on the application protocol using QUIC. Making this a public function is dangerous as it might encourage bad practice. 

If you'd like a convenience or utility function to make the tests easier, please put a config creation function inside the test module